ledger-api/grpc-definitions/com/daml/ledger/api/v1/commands.proto

```diff
@@ -217,9 +218,17 @@ message DisclosedContract {
   // The contract id
   // Required
   string contract_id = 2;
+
   // The contract arguments
-  // Required
-  Record arguments = 3;
+  oneof arguments {
+    // The contract arguments as typed Record
+    Record record = 3;
```

Comment:
This change is binary compatible, so far so good
It is slightly annoying if you absorb an sdk upgrade, recompile the protos and now have to change potentially many places in your code due to a name change. How about
```
oneof contract_arguments {
  Record arguments = 3;
  google.protobuf.Any blob = 5;

Reply:
But if we were to go for a field name change, I would make it consistent with the receiving side i.e.:
```
oneof arguments {
  Record create_arguments = 3;
  google.protobuf.Any create_arguments_blob = 5;
```